javaslang/src/main/java/javaslang/collection/Seq.java

```diff
@@ -139,6 +148,50 @@ default T apply(Integer index) {
     Seq<T> appendAll(Iterable<? extends T> elements);
 
     /**
+     * Creates an <strong>immutable</strong> {@link java.util.List} view on top of this {@code Seq},
+     * i.e. calling mutators will result in {@link UnsupportedOperationException} at runtime.
+     * <p>
+     * The difference to conversion methods {@code toJava*()} is that
+     *
+     * <ul>
+     * <li>A view is created in O(1) (constant time) whereas conversion takes O(n) (linear time), with n = collection size.</li>
+     * <li>The operations on a view have the same performance characteristics than the underlying persistent Javaslang collection whereas the performance characteristics of a converted collection are those of the Java standard collections.</li>
+     * </ul>
+     *
+     * @return A new immutable {@link java.util.Collection} view on this {@code Traversable}.
+     */
+    java.util.List<T> asJava();
```

Comment:
we can't define Traversable.asJava() that returns j.u.Collection&lt;T> because j.u.Map does not extend Collection.